tracer/src/Datadog.Trace/Datadog.Trace.csproj

```
@@ -102,7 +102,7 @@
     <DefineConstants>$(DefineConstants);ATOMIC_APPEND;HRESULTS</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1'">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1' OR '$(TargetFramework)' == 'net6.0'">
```

More future proofing
```suggestion
  <PropertyGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework'">
```